README.md

```diff
@@ -16,45 +80,114 @@ Generating this manifests is very easy, you just need add the proper dependency
  </dependency>
 ```
 
-We also need to add an annotation to enable dekorate. In this case we will use `@KubernetesApplication` which also gives us access to more Kubernetes specific configuration options.
-Edit the App class and add the indicated annotation.
+We need to enable `Dekorate` by adding an annotation. In this case we will use `@KubernetesApplication` which also gives us access to more Kubernetes specific configuration options.
+Edit the App class and add the following annotation.
 
 ```
-@KubernetesApplication(name = "hello-world-fwless-k8s", ports = @Port(name = "web", containerPort = 8080))
+@KubernetesApplication(
+        name = "hello-world-fwless-k8s",        
+        ports = @Port(name = "web", containerPort = 8080),  
+        host = "fw-app.127.0.0.1.nip.io", //2
+        expose = true, //3
+        imagePullPolicy = ImagePullPolicy.Always //4
+)
 ```
-We need to specify a port in order to prevent dekorate it is about a Web Application, this way a service will be generated in the k8s manifest.
+- We need to specify a **`@Port`** in order to prevent dekorate it is about a Web Application, this way a service will be generated in the k8s manifest.
 
-NOTE: if you are using minikube a trick that often works is to use a service of type NodePort.
```

Comment:
this trick is not specific to minikube and could be used with kind, minishift, ... ;-)